animation_nodes/nodes/matrix/distribute_matrices.pyx

```diff
@@ -249,6 +259,33 @@ class DistributeMatricesNode(bpy.types.Node, AnimationNode):
 
         return matrices
 
+    def execute_Fibonacci(self, Py_ssize_t amount, Py_ssize_t centerMask, float angle, float startSize, float endSize, float scale):
+        centerMask = min(centerMask, amount)
+
+        cdef Py_ssize_t i = centerMask
+        cdef float factor = 1 / <float>((amount - centerMask) - 1) if (amount - centerMask) > 1 else 0
+        cdef float goldenAngle = angle * (PI / 180)
+        cdef float r, theta, size, f
+        cdef Vector3 vector
+        cdef Matrix4x4List matrices = Matrix4x4List(length = amount - centerMask)
+
+        while i < amount:
+            f = <float>(i - centerMask) * factor
+            size = f * (endSize - startSize) + startSize
+            
+            theta = i * goldenAngle
+            r = sqrt(i) / sqrt(amount) * scale
+
+            vector.x = cos(theta) * r
+            vector.y = sin(theta) * r
+            vector.z = 0
```

Comment:
Can we adjust the code to avoid computing sine the cosine inside the loop? The spiral code we have already does this by incrementally rotating some initial point. See the 2D rotation matrix as a reference. Let me know if you need more details.

Reply:
@OmarEmaraDev  Yes need bit more explanation. I didn't understand it's working. or an example maybe helpful